### DISCUSSION TITLE
Charging Battery

### DISCUSSION THREAD

**Discussion by immortal21god** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#123] [Q&A]
Can someone please tell me how I should charge my battery, whilst utilising AlDente? I’ve currently set my charging limit to 60% because apparently that was equivalent to around 3.92V which was the best condition the battery could ask for. Do I also have to drain my battery to 0% every month and charge it back to 100%? What is the significance of this?

**Comment by KissDanielMark** [comment]
I've read that it is recommended to do a full 100%->0% cycle every month. Although there are lots of articles says it is not necessary...
I started using this app yestarday because Apples own battery saver algorithm doesn't work and it keeps my battery at 100% all the time. In the past (roughly 2 years) I always used my laptop on charger and this week I had to bring it to a service becasue my battery got humped. My battery had only 100 cycles. Hope this will prevent it not to happen again.

**Reply by immortal21god** [reply]
I myself have bought the 16 inch a few months ago and I don't want to pay for AppleCare+ since it's £399 so I can understand where you're coming from.

**Comment by Limeybastid** [comment]
Personally, I would periodically charge it past the limit you have Al Dente set to, I don't believe it needs to be drained to zero at any time.
